### ISSUE TITLE
Should I use v4992?

### ISSUE THREAD

**Issue by AdminUser0** [pleriche/FastMM4#84]
Hi!
I downloaded some  years ago from here v4992. Now this version is not available for download. Was it a beta version? Should I keep using it?
Thanks.

**Comment by JensMertelmeyer** [comment]
4.992 is, in fact the only version available for download:
https://github.com/pleriche/FastMM4/releases/tag/release_4992

A few days before your post, the source code incorrectly still had `4.991`, see https://github.com/pleriche/FastMM4/issues/79

**Comment by idanmiara** [comment]
Hi @pleriche 
I see that 4.992 (from 10/2016) is the only released version.
Is Master stable (latest commit was on 4/2020)? Could a new version be released/tagged ?
Thanks!

**Comment by pleriche** [comment]
Yes, master is stable. New development work is going into version 5. I have made a new release, 4.993, that contains all the improvements and fixes of the last few years.

**Comment by idanmiara** [comment]
> Yes, master is stable. New development work is going into version 5. I have made a new release, 4.993, that contains all the improvements and fixes of the last few years.

Thanks :)
